packages/model-typings/src/models/IRoomsModel.ts

```diff
@@ -239,7 +239,7 @@ export interface IRoomsModel extends IBaseModel<IRoom> {
 	incUsersCountById(rid: string, inc: number): Promise<UpdateResult>;
 	incUsersCountNotDMsByIds(rids: string[], inc: number): Promise<Document | UpdateResult>;
 	setLastMessageById(rid: string, lastMessage: IRoom['lastMessage']): Promise<UpdateResult>;
-	resetLastMessageById(rid: string, lastMessage?: IMessage | null): Promise<UpdateResult>;
+	resetLastMessageById(rid: string, lastMessage?: IMessage | null, msgCounter?: number): Promise<UpdateResult>;
```

Comment:
```suggestion
	resetLastMessageById(rid: string, lastMessage?: IMessage | null, msgCountDelta?: number): Promise<UpdateResult>;
```
Just trying to make the param's purpose clearer